free-3.md

```diff
@@ -1,43 +1,52 @@
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it streamlines the process for certain users. This DSL definitely appeals to a niche category of users that much prefer function over form. As someone always on the computer, it would be nice if I could simply schedule an event within a few seconds rather than having to open a browser, go to the calender app, and add it. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I know that I am a forgetful person and would benefit exceptionally from a planner or calender of some sort; however, I find myself too lazy to actually go out of the way to create an event or reminder in Google Calenders or iCal, especially if the event is within a few days or even a few hours. It's ironic because it is these events that tend to be the most important. What excites me is the idea of simply oepning my computer, typing a command in terminal or some other environment and have the reminder automatically created.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Scheduling made for lazy people
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would simply type a series of commands that would compile into a .csv format. I imagine the program would be very simplistic and similar to terminal commands, something along the lines of "schedule: 3/20, 9pm - 10pm, hwk" or "schedule: 1/6, 1pm - 4pm, e59 class, repeat until 5/1." I think this is the right way to interact with scheduling because something made with the intention of improving time management shouldn't waste time.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs, the commands are compiled into a valid .csv format that can be easily imported into a calender app. The program interacts with the user by taking their commands, which are in relatively readable code, and making them nicely formatted. Some errors that might occur include not filling out required fields or messing up the sequence of arguments, since (based off of the above example) it's pretty arbitrary right now.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+The expressiveness is super limited for this language, as the domaind of "scheduling" isn't particularly extensive. It should be easy to schedule! It should be impossible or very very difficult to do anything else, really. 
 
```

Comment:
You might want to expand this to types of scheduling that would be possible or difficult. For example, it should perhaps be possible to schedule conflicting or identical events. The DSL might make it difficult with a confirmation prompt. It should perhaps be difficult (or impossible for the time being) to interact with or view other calendars. Just some suggestions!